docs/user_guide/mitigation.rst

```diff
@@ -81,6 +94,707 @@ Postprocessing
+    ...                           predict_method='predict_proba')\
+    ...                           .fit(X_train,Y_train,sensitive_features=sf_train)
+    
+    >>> Y_pred_switched_threshold = thresholder.predict(X_test,sensitive_features=sf_test)
+
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=Y_pred_switched_threshold, 
+    ...             sensitive_features=sf_test).by_group
+    race
+    AfricanAmerican    0.381274
+    Asian              0.878049
+    Caucasian          0.399399
+    Hispanic           0.365217
+    Other              0.875000
+    Name: false_negative_rate, dtype: float64
+
+    >>> plot_positive_predictions(sf_test,Y_pred_switched_threshold)
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_switch_predictions.png
+    :align: center
+
+Specifying a default threshold
+********************************
+In the previous examples, the threshold used for the groups that were not
+specified in the :code:`threshold_dict` was the default threshold of 0.5 (if
+:code:`predict_method` is either :code:`predict_proba` or :code:`predict`). It
+is possible to specify a different default threshold using the
+:code:`default_threshold` parameter. This threshold can be specified using all
+ways shown above.
+
+.. doctest:: mitigation
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> threshold_dict = {'Asian': 0.4, 'Other': ('>', 0.4)}
+    >>> thresholder = Thresholder(estimator=classifier,
+    ...                           threshold_dict=threshold_dict,
+    ...                           prefit=True,
+    ...                           predict_method='predict_proba',
+    ...                           default_threshold=0.6)\
+    ...                           .fit(X_train,Y_train,sensitive_features=sf_train)
+    
+    >>> default_thresholded_pred = thresholder.predict(X_test,sensitive_features=sf_test)
+
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=default_thresholded_pred, 
+    ...             sensitive_features=sf_test).by_group
+    race
+    AfricanAmerican    0.672780
+    Asian              0.121951
+    Caucasian          0.673135
+    Hispanic           0.704348
+    Other              0.125000
+    Name: false_negative_rate, dtype: float64
+
+    >>> plot_positive_predictions(sf_test,default_thresholded_pred)
+
+.. figure:: ../auto_examples/images/user_guide_thresholder_default_threshold_0_6.png
+    :align: center
+
+Dealing with the Prefit parameter
+**********************************
+In all examples above, the :code:`prefit` parameter was set to :code:`True`.
+This parameter indicates whether or not fit has been called on the supplied
+estimator. If :code:`Prefit==False`, the :code:`Thresholder.fit()` function
+will fit the estimator. For the final predictions it does not
+matter whether or the estimator is fit beforehand or in the 
+:code:`Thresholder.fit()` call:
```

Comment:
Yeah I wasn’t sure how detailed I should go into the different parameters, so I added this to be sure. I can see how it might be a bit redundant. I’ll remove it for now.